.eslintrc

```diff
@@ -9,12 +9,32 @@
   ],
   "plugins": ["jsx-a11y"],
   "rules": {
+
     // driving me nuts and would have involved changing a zillion files for no real benefit
     "react/jsx-one-expression-per-line": "off",
     "quotes": "off",
     "semi": "off",
     "camelcase":"off",
     "no-unused-expressions": "off",
```

Comment:
what is the difference to line 34 (vars)

Reply:
Can we also remove the "Driving me nuts" line, it's an old comment from @DurkoMatko